src/store/server/index.js

```diff
@@ -35,7 +35,7 @@ axiosInstance.interceptors.response.use(
   }
 );
 
-async function getRecord(url) {
+export async function getRecord(url) {
```

Comment:
To use get record, there is a findRecord function which you can use.

Reply:
ok trideep ill look at it.